applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.h

```diff
@@ -151,6 +152,28 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ElasticIsotropic3D
         return StressMeasure_Cauchy;
     }
 
+    /**
+     * @brief Adds the initial stress vector if it is defined in the InitialState
+     */
+    const void AddInitialStressVectorContribution(Vector& rStressVector, Parameters& rParameterValues) 
```

Comment:
cannot those functions be `static` functions of the `ConstitutiveLaw`?
then they don't have to be reimplemented in every element and this would probably also solve the downcasting issue

Reply:
I've tried and I've noticed that inside this method there's a return of a non static `mpInitialState `so they cannot be static I think

Reply:
Ok then without the static would be working I think